buildSrc/src/main/kotlin/com/strumenta/kotlinmultiplatform/gradle/ext/Project.ext.kt

```diff
@@ -47,12 +39,29 @@ fun Project.booleanProperty(name: String): Boolean =
 /**
  * Returns whether we are building for a release, or not.
  */
-fun Project.releaseBuild(): Boolean =
-  !(stringProperty("version")?.endsWith("-SNAPSHOT")
-    ?: throw IllegalStateException("Project version not specified"))
+fun Project.releaseBuild(): Boolean {
+  val isSnapshot = stringProperty("version")?.endsWith("-SNAPSHOT")
+    ?: throw IllegalStateException("Project version not specified")
+
+  return !isSnapshot
+}
+
+/**
+ * Returns the name of the private Maven repository to publish artifacts to,
+ * or `null` if none is configured for the given publishing policy (release/snapshot).
+ */
+fun Project.mavenRepositoryName(): String? {
```

Comment:
I am not sure if we need to make this configurable because I think we want always to publish to Maven Central. Do you think some companies may want to publish to some internal Nexus?

Reply:
I do for example, and it might definitely comes handy having it "pre-configured", with a developer only having to configure a couple `gradle.properties` values.